src/components/Input/Input.spec.js

```diff
@@ -25,6 +25,13 @@ describe('Input Component', () => {
     expect(wrapper.find('input')).toHaveLength(1);
   });
 
+  it('renders an error below when it recieves error props', () => {
+    const { wrapper } = setup({ error: 'first name is required' });
+
+    expect(wrapper.exists()).toBeTruthy();
+    expect(wrapper.find('.input-validation-error')).toHaveLength(1);
```

Comment:
you're supposed to initially check that `input-validation-error` is not present before the error, also check that `input-validation-error` contains error that occurred

Reply:
@dinobi I have created a new test case to check that that `input-validation-error` is not present where there is nor error prop passed.

I have also asserted that the text content of `input-validation-error` is the same as the value of the error prop that was passed.